### ISSUE TITLE
Add tabindex to text elements to make them keyboard accessible

### ISSUE THREAD

**Issue by v-licope** [cebor/angular-highcharts#342]
Users should be able to access the tooltips in the chart with the keyboard instead of mouse for accessibility - text elements should have a tabindex to make this happen - right now this is not the case - examining the highcharts.js site I see that there is an accessibility module - are you guys using this? I think it would solve this issue
